.github/workflows/codeql.yml

```diff
@@ -78,7 +78,7 @@ jobs:
     - if: matrix.language == 'cpp'
       name: Build C++
       run: |
-        CMAKE_BUILD_TYPE=Debug PLATFORM=virtual BUILD_TESTS=OFF ./build.sh
```

Comment:
From a quick look at build.sh, the default seems to have changed to Debug, and so the behaviour is still the same I guess?

Reply:
it is all the same I just moved variables to a preset as it is being used all over the place over and over again, not to mention that prod builds happen in a container anyway and this is more of a development/debug/ci script